packages/styling/src/interfaces/ISemanticColors.ts

```diff
@@ -6,6 +6,11 @@ import { ISemanticTextColors } from './ISemanticTextColors';
  * The collection of all semantic slots for colors used in themes.
  *
  * Note: text colors are defined in ISemanticTextColors.ts.
+ * The plan is to move completely to semantic slots and replace all usage of fabric palette slots.
```

Comment:
"The plan" is a bit vague and informal. "We plan to..." or "We hope to...".
"Fabric" should be capitalized.